packages/wxt/src/core/utils/building/generate-wxt-dir.ts

```diff
@@ -31,7 +32,8 @@ export async function generateTypesDir(
     const unimport = createUnimport(wxt.config.imports);
     references.push(await writeImportsDeclarationFile(unimport));
     if (wxt.config.imports.eslintrc.enabled) {
-      await writeImportsEslintFile(unimport, wxt.config.imports);
+      const eslintVersion = await getEslintVersion();
+      await writeImportsEslintFile(unimport, wxt.config.imports, eslintVersion);
```

Comment:
We can just move this variable into the function

Reply:
moved! :)